lint/action.yaml

```
@@ -1,30 +1,29 @@
 name: "terraform Lint"
 description: "Lints terraform repository"
 inputs:
-  checkout-repo:
+  checkout_repo:
     required: false
     description: "Perform checkout as first step of action"
     default: true
-  github-token:
+  github_token:
     required: true
     description: "GitHub token that can create/delete comments. Usually - 'secrets.GITHUB_TOKEN'"
+  atlas_token:
+    required: true
+    description: "The Terraform enterprise atlas token"
 runs:
   using: composite
   steps:
     - uses: actions/checkout@v2
-      if: inputs.checkout-repo
+      if: inputs.checkout_repo
       with:
         fetch-depth: 0
+    - uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 1.0.0
```

This should probably be 1.1.2 - just to match our current version.  until we fix things with `action-setup-tools`
```suggestion
        terraform_version: 1.1.2
```

---

This should be addressed.